deploy/kubernetes/keytransparency-deployment.yml.tmpl

```diff
@@ -46,11 +48,15 @@ metadata:
   labels:
     run: kt-server
 spec:
-  type: NodePort
+  type: LoadBalancer
+  loadBalancerIP: 35.184.134.53
   ports:
     - port: 8080
       targetPort: 8080
       name: json-grpc
+    - port: 8081
```

Comment:
Done: Now there is an external service for the kt-server (that exposes only the web interface to the world) and to "internal" (type: NodePort) services, such that prometheus can find the dns `kt-server` and `kt-signer`. NodePort ports do not autm. create a firewall rule that expose the port (while LoadBalancer type services do).